Objects/Converters/ConverterRevit/ConverterRevitShared/RevitCommitObjectBuilder.cs

```diff
@@ -92,13 +86,13 @@ public override void IncludeObject(Base conversionResult, Element nativeElement)
     switch (_commitCollectionStrategy)
```

Comment:
Is there only one of these? I cannot see what the switch is adding here?

Reply:
resolving as autoformatting change